dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/net/RumOkHttpUploader.kt

```diff
@@ -16,6 +18,12 @@ internal open class RumOkHttpUploader(
     client: OkHttpClient
 ) : DataOkHttpUploader(buildUrl(endpoint, token), client, CONTENT_TYPE_TEXT_UTF8) {
 
+    private val tags: String by lazy {
+        arrayOf(
+            "${LogAttributes.SERVICE_NAME}:${CoreFeature.serviceName}",
```

Comment:
You should create the same fields in RumAttributes. 
Because of the renaming of various fields (cf data model v2), we should only use  Rum specific constants in RUM package, and same for logs